test/support/conn_case.ex

```diff
@@ -22,6 +22,8 @@ defmodule DemoWeb.ConnCase do
       # Import conveniences for testing with connections
       import Plug.Conn
       import Phoenix.ConnTest
+      import DemoWeb.ConnCase
```

Comment:
@aaronrenner I have changed Phoenix v1.5 generators to automatically import the case itself, so we won't need this change in our generators themselves.